docs/antrea-network-policy.md

```diff
@@ -409,6 +411,58 @@ spec:
         enableLogging: true
 ```
 
+#### ACNP for IGMP traffic
+
+```yaml
+apiVersion: crd.antrea.io/v1alpha1
+kind: ClusterNetworkPolicy
+metadata:
+  name: acnp-with-igmp-drop
+spec:
+    priority: 5
+    tier: securityops
+    appliedTo:
+      - podSelector:
+          matchLabels:
+            app: mcjoin6
+    ingress:
+      - action: Drop
+        protocols:
+        - igmp:
+            igmpType: 0x11
+            groupAddress: 224.0.0.1
+        name: dropIGMPQuery
+    egress:
+      - action: Drop
+        protocols:
+        - igmp:
+            igmpType: 0x16
+            groupAddress: 225.1.2.3
+    name: dropIGMPReport
+```
+
+#### ACNP for multicast egress traffic
+
+```yaml
+apiVersion: crd.antrea.io/v1alpha1
+kind: ClusterNetworkPolicy
+metadata:
+  name: acnp-with-multicast-traffic-Drop
```

Comment:
name doesn't allow upper case, please verify if the policy really works

Reply:
done